### DISCUSSION TITLE
How do I replicate the default fan color scheme?

### DISCUSSION THREAD

**Discussion by benjj-wr** [devcompl/TTController#113] [Configuration] [answered]
Hello again,
I wanted to ask how I can replicate the default fan color scheme? To clarify, I attached a video, but it is kinda hard to see what I mean. It is basically a 12 color rainbow that is moving in a circular motion. I tried all of the effects but was not able to replicate it to look exactly like that. Maybe someone can help me out. Thanks so much in advance!

https://user-images.githubusercontent.com/76590646/103364467-2cc0b300-4abe-11eb-96a1-19c17e8be0b3.mp4

**Comment by MoshiMoshi0** [comment, marked as answer]
`AuroraEffect` with `"Length"` set to 12 should be close.

**Reply by benjj-wr** [reply]
Exactly what I was looking for. Thank you so much.
Just donated 10 bucks. I know it's not much, but I wanted to give back at least a little. Have a happy new year!
